src/harness/testcases/heartbeat_testcase.py

```diff
@@ -78,6 +78,198 @@ def test_10_9_4_1_1_1(self):
     self.assertEqual(response['response']['responseCode'], 0)
 
   @winnforum_testcase
+  def test_WINNF_FT_S_HBT_2(self):
+    """Multiple heartbeat requests after moving to Granted/Heartbeating state.
+
+    Returns response code 0 (NO_ERROR) for all requests
+    """
+    # Register three devices
+    device_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    self._sas_admin.InjectFccId({'fccId': device_a['fccId']})
+    device_b = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    self._sas_admin.InjectFccId({'fccId': device_a['fccId']})
+    device_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    self._sas_admin.InjectFccId({'fccId': device_a['fccId']})
+    request = {'registrationRequest': [device_a, device_b, device_c]}
+    response = self._sas.Registration(request)['registrationResponse']
+    cbsd_ids = []
+    for resp in response:
+      self.assertEqual(resp['response']['responseCode'], 0)
```

Comment:
Our formatting tool always uses 2 spaces for indenting into functions and loops. However, other issues should be ironed out now